app/lib/authentication/by_cookie_token.rb

```diff
@@ -51,11 +51,10 @@ def refresh_token
       # always be deleted together.
       #
       def forget_me
-        self.remember_token_expires_at = nil
-        self.remember_token            = nil
```

Comment:
Shouldn't we still set these?

Reply:
Isn't it done by the update? If not... we should probably test it. Somehow.

Reply:
I don't think `#update_clomuns` will update current object.

Reply:
maybe it does

Reply:
Ah, apparently it does. I'll add a test for this